example/capacitor.config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
necessary or leftover from your local setup?

Reply:
This was autogenerated, as far as I can tell.

Reply:
Can you add this to .gitignore instead of committing it?

Reply:
Let me delete it and see whether it gets recreated.

Reply:
This file is for configuring the app, and isn't just machine-generated, so it shouldn't be in `gitignore`.

Reply:
Maybe drop the comment? For me, running this command for android allows me to select the device, then launches the app on the emulator or real device. Technically, I have AS running already, so it could just be that it relies on the individual SDKs to perform the actual deploy.